server/services/notificationService.js

```diff
@@ -1,6 +1,6 @@
 const templates = require('./config/notificationTemplates')
 const notificationMailboxes = require('./config/notificationMailboxes')
-const { notifyKey } = require('../config').notifications
+const { notificationConfig, domain } = require('../config')
```

Comment:
would be better to destructure this instead i.e. something like
```const { notifications: { notifyKey }, domain } = require('../config')
```
and then you can just refer to the `notifyKey` as before

Reply:
nice, ta